trace/opencensus/trace/utils.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import uuid
+
+
+def generate_trace_id():
```

Comment:
Would it make sense to put generate_trace_id() and generate_span_id() in span_context.py?  Related, from the discussion yesterday, should we remove span_id from Span, and have Span contain a SpanContext?

Reply:
Moved to `span_context.py`. Yes will move SpanContext to Span in later PR.